lib/install/opsuser/opsuser.go

```diff
@@ -98,12 +101,27 @@ func (mgr *RBACManager) SetupDCReadOnlyPermissions(ctx context.Context) (*rbac.R
 	if err != nil {
 		return nil, err
 	}
+
 	// If administrator, skip setting the root permissions
 	if res {
 		log.Warnf("Cannot perform ops-user Role/Permissions initialization. The current ops-user (%s) has administrative privileges.", am.Principal)
 		log.Warnf("This occurs when \"%s\" is a member of the \"Administrators\" group or has been granted \"Admin\" role to any of the resources in the system.", am.Principal)
 		return nil, errors.Errorf("Cannot grant ops-user permissions as %s has administrative privileges", am.Principal)
 	}
+
+	dcRef := mgr.session.Datacenter.Reference()
+	hasPrivs, err := am.ReadOnlyOnRef(ctx, dcRef)
+	if err != nil {
+		return nil, err
+	}
+
+	// If the ops-user already has enough privileges on the datacenter to at least
+	// satisfy the read-only role, skip setting read-only permissions.
+	if hasPrivs {
+		log.Infof("ops-user already has enough privileges for read-only access to datacenter, skipping setting read-only permissions")
```

Comment:
This is a debug message - we absolutely don't need to be telling the caller that we're _not_ doing steps.

Reply:
That makes sense. If we're not altering a user's permissions it's perfectly fine to log this at the debug level. cc @matthewavery